package.json

```
@@ -32,7 +32,8 @@
     "prebuild:tokens": "rm -rf tokens-v2-private",
     "prepack": "yarn build && yarn build:tokens",
     "release": "changeset publish",
-    "watch": "ls data/**/*.scss script/**/*.ts | entr -s 'yarn build'"
+    "watch": "ls data/**/*.scss script/**/*.ts | entr -s 'yarn build'",
+    "test": "jest"
```

Totally optional, but I like running this with the following flags `--verbose --coverage` because it gives additional context into what ran and how tests performed..

```suggestion
    "test": "jest --verbose --coverage"
```